client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -161,16 +160,18 @@ class TransferOtherUser extends React.Component {
 			return <DomainMainPlaceholder goBack={ this.goToEdit } />;
 		}
 
-		const { selectedSite, selectedDomainName, currentRoute } = this.props,
-			{ slug } = selectedSite;
+		const { selectedSite, selectedDomainName, translate, currentRoute } = this.props;
+		const { slug } = selectedSite;
 
 		return (
 			<Main>
 				<Header
 					selectedDomainName={ selectedDomainName }
 					backHref={ domainManagementTransfer( slug, selectedDomainName, currentRoute ) }
 				>
-					{ this.props.translate( 'Transfer Domain To Another User' ) }
+					{ this.props.isMapping
+						? translate( 'Transfer Domain Mapping To Another User' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.